.github/workflows/prebuild.yml

```
@@ -69,3 +71,18 @@ jobs:
       with:
         name: ${{ matrix.platform }}-${{ matrix.arch }}${{ matrix.tags }}
         path: prebuilds/*
+  zip:
+    needs: prebuild
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download prebuilds
+        uses: actions/download-artifact@v4
+        with:
+          path: prebuilds
+      - name: Zip prebuilds
+        run: zip -r prebuilds.zip prebuilds/
+      - name: Upload zip
+        uses: actions/upload-artifact@v4
+        with:
+          name: prebuilds
+          path: prebuilds.zip
```

```suggestion
  zip:
    needs: prebuild
    runs-on: ubuntu-latest
    steps:
      - uses: actions/download-artifact@v4
        with:
          path: prebuilds
      - uses: actions/upload-artifact@v4
        with:
          name: prebuilds
          path: prebuilds
```

This should suffice and also avoid the double archive.